app/localization.py

```diff
@@ -90,5 +90,49 @@
 Обрати внимание, что ты можешь использовать меня в любом чате через inline-режим!
 """
     }
+    'pt-BR': {
```

Comment:
How much is PT-BR different from PT-PT? Maybe, is it better to use more general `'pt'` for the localization?

Reply:
Let me check if there will be any difference.

Reply:
There are some differences, but nothing critical. Will update the file.